src/test/groovy/edu/ie3/datamodel/io/naming/FileNamingStrategyTest.groovy

```diff
@@ -859,7 +859,7 @@ class FileNamingStrategyTest extends Specification {
 		def path = Paths.get(pathString)
 
 		when:
-		def metaInformation = fns.extractTimeSeriesMetaInformation(path)
+		def metaInformation = fns.timeSeriesMetaInformation(path)
```

Comment:
Please consider renaming of `fns` to `ens` or something more meaningful (throughout the whole test).

Reply:
Well here, it is actually a `FileNamingStrategy`